### DISCUSSION TITLE
Is there a ControlNet + T2I Adapter Pipeline available for Stable Diffusion 1.5?

### DISCUSSION THREAD

**Discussion by whiteprimrose** [huggingface/diffusers#9287] [Q&A] [answered]
Hello everyone,

I've been exploring the community examples in the Diffusers library and came across the [ControlNet + T2I Adapter Pipeline](https://github.com/huggingface/diffusers/tree/main/examples/community#controlnet--t2i-adapter-pipeline), which is fantastic for Stable Diffusion XL (SDXL).

However, I am currently working with Stable Diffusion 1.5 (SD1.5) and would like to implement similar functionality. Does anyone know if there is an equivalent pipeline or adaptation method available for SD1.5?

Any guidance or pointers to existing implementations or discussions would be greatly appreciated.

Thank you!

**Comment by asomoza** [comment, marked as answer]
HI, there isn't a equivalent pipeline for SD, the one for SDXL was contributed by @affromero in this [PR](https://github.com/huggingface/diffusers/pull/5869).

Since SD is kind of old (I mean no new controlnets and T2I Adapters for a while) I don't think there's that much people willing to contribute to this. Maybe you want to try opening a PR with this? You can use the SDXL one as a guide and for SD it should be easier.

**Reply by whiteprimrose** [reply]
Thanks for your response. I appreciate the help.
